lib/index.js

```diff
@@ -3,7 +3,7 @@ const path = require('path');
 const BinWrapper = require('bin-wrapper');
 const pkg = require('../package.json');
 
-const url = `https://raw.github.com/imagemin/pngquant-bin/v${pkg.version}/vendor/`;
+const url = `https://raw.githubusercontent.com/imagemin/pngquant-bin/v${pkg.version}/vendor/`;
```

Comment:
should this be in a `encodeURI()` call?

Reply:
@shaun-sweet `encodeURI()` has no sense here, it does not encode specials characters like `:`, `.` or `/` that can be found in this URL, please do not confuse with `encodeURIComponent()`. Please the the [corresponding article](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURI) on MDN.

Reply:
Good call!!